lib/default-theme/NavLinks.vue

```diff
@@ -69,14 +69,22 @@ export default {
         })
       }))
     },
-    githubLink () {
+    repoLink () {
       const { repo } = this.$site.themeConfig
       if (repo) {
         return /^https?:/.test(repo)
           ? repo
           : `https://github.com/${repo}`
       }
-    }
+    },
+    repoLabel () {
+      if (this.$site.themeConfig.repoLabel) return this.$site.themeConfig.repoLabel
+
+      const repoHost = this.repoLink.match(/^https?:\/\/[^/]+/)[0] || 'github'
```

Comment:
Since repo link start with `https://`, so the check `match(/^https?:\/\/[^/]+/)[0]` is duplicated. 

in other words, the statements `match(/^https?:\/\/[^/]+/)[0]` is dangerous since `match` would return undefined!

Reply:
This is true. Will fix this later.